README.md

```diff
@@ -167,5 +169,33 @@ slides:
   caption: >
     I often try, actually
 ```
+### Theme
+To get started using the provided 
+default theme, add the `slides-theme` 
+bundle to your project:
+
+```html
+<style @raw="getBundle('css', 'slides-theme')" webc:keep></style>
+```
+
+The theme in this layer adds settings for colors, basic styling,
+and fonts. 
+
+In the `slides-theme` layer we list 
+default font stacks for sans-serif 
+(`--slide-os-sans-font`), serif (`--slide-os-serif-font`), 
+and code (`--slide-os-code-font`) fonts. 
+
+To add a main font for each font stack type, manually 
+add self-hosted or web-based fonts to your project and then set the
+font-family name to the respective custom property for each font. 
+
+Example:
+
+```css
+--slide-web-font-sans: "Recursive Sans Linear";
+--slide-web-font-serif: freight-text-pro;
+```
```

Comment:
I think this is great, can we also set it up with our fonts in the test site, to make sure it all works as expected?

Reply:
Added the fonts and set these properties in https://github.com/oddbird/eleventy-plugin-slide-deck/pull/2/commits/af04baea5393f6b10a1c1aa47ca6fa366b870af3. It seems like everything is working as expected.